### ISSUE TITLE
synonyms.sparql query

### ISSUE THREAD

**Issue by zhengj2007** [INCATools/ontology-development-kit#147]
Can also retrieve annotation property, alternative term (IAO_0000118), as synonyms? So, it can work with ontologies that using IAO annotation property.

**Comment by cmungall** [comment]
OK, I wonder if this should all be moved into robot as we do with qc reports

**Comment by zhengj2007** [comment]
Good suggestion.

**Comment by matentzn** [comment]
@zhengj2007 can you create the ticket on [robot](https://github.com/ontodev/robot/issues) and close it here? Thanks :)

**Comment by jamesaoverton** [comment]
@cmungall You're suggesting moving all of [template/src/sparql](
https://github.com/INCATools/ontology-development-kit/tree/master/template/src/sparq) right? That's much larger scope than what @zhengj2007 is asking for.

**Comment by cmungall** [comment]
yes, would be good to have standard reports for aggregate statistics and standard TSVs IMO
